### PR TITLE
Feat: Added cards

### PR DESCRIPTION
**Pull request by LordUttam**
Added cards component. Please review, thanks.

### CODE REVIEW

File: styles/cards.css [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+.card__header {
+  display: flex;
+  flex-direction: column;
+  gap: 2px;
+}
+
+.card__title {
+  margin-bottom: 0px;
+  padding: 0 1rem;
```
Comment (sonishreyas) [inline styles/cards.css:36]: you can use variables for margin and padding it helped me.

**Review by sonishreyas — APPROVED** [review]
Looks good to me

**Comment by parthpandyappp** [comment]
LGTM as well :+1:

File: styles/cards.css (hunk 1 above)
Comment (LordUttam) [reply styles/cards.css:36]: Thanks for the suggestion.

**Pull request merged**